### PR TITLE
Customer promise

### PR DESCRIPTION
**Pull request by benbarnett**
Linked to from the support links in the footer.

### CODE REVIEW

File: content/customer_promise.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- Continuously improve our services to make things better for you
+
+##Accessibility
+
+- Make sure you can get a face-to-face appointment if you have a disability -- eg wheelchair access or in British Sign Language if you’re deaf
+- Send your summary in braille or large print if you’re blind or partially sighted
+
+^In return we need you to:^
+
+- be on time for your appointment or let us know if it needs to change
```
Comment (benlovell) [inline content/customer_promise.md]: Should these be capitalised per the other bullets?

Comment (benbarnett) [inline content/customer_promise.md]: Potentiado – @alexkouz what say you?

Comment (benbarnett) [inline content/customer_promise.md]: Looking through the rest of the site, capitalising the first word would keep this consistent. I took the liberty of changing it here too.

Comment (alexkouz) [inline content/customer_promise.md]: You mean the bullets under 'In return...'? Yes, please cap. Ta.

**Pull request merged**